src/components/outline-nav/props.js

```diff
@@ -0,0 +1,33 @@
+module.exports = {
```

Comment:
Oh man swingset 😭😎 Can we add a breadcrumb on the type file to keep these two in sync? 🍞

Reply:
I think I'd rather see us link to the `types.ts` file from the Swingset page rather than have multiple things to maintain with the component.

Reply:
ah yep... added a comment! and linked to 🎟 [Support extracting props from TS](https://app.asana.com/0/1199715139452823/1199702038395513/f).

Reply:
ah oops Ashlee i didn't see your comment - we could link to `types.ts`, but then we wouldn't get the props table on the Swingset page. Happy to do either, if we want to avoid using `props.js` and the props table in Swingset, i can go that route 👍

Reply:
I don't feel strongly that we need to have the props table on the Swingset page. I don't think we've done it much yet?